go-controller/pkg/ovn/policy_test.go

```diff
@@ -2311,3 +2372,9 @@ var _ = ginkgo.Describe("OVN NetworkPolicy Low-Level Operations", func() {
 		gomega.Expect(gp.delNamespaceAddressSet(four.GetObjectID(libovsdbops.ObjectNameKey))).To(gomega.BeFalse())
 	})
 })
+
+func buildNetworkPolicyAddressSets(namespaceName string, peer knet.NetworkPolicyPeer, ips ...string) (*nbdb.AddressSet, *nbdb.AddressSet) {
```

Comment:
I want to change all the build address set functions to this format taking variadic arguments this is just the first one to get this treatment

Reply:
But is the PR ready for review ? Will you be making those changes in this PR ?

Reply:
my plan was to make those in future PRs.

Reply:
I'll review this one then ASAP. Thank you.